cmput_404_project/posts/templates/posts/private_posts.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Posts</title>
+    </head>
+    <body>
+        <h1>List of Private Posts</h1>
+        <div class="posts">
+            {% for post in posts %}
+            <div class="post">
+                <p> <b> Published: </b> {{post.published}} </p>
+                <p> <b> Author: </b> {{post.author}} </p>
+                {% if post.image != null and post.image != '' %}
+                    <img src="{{post.image}}"> 
+                {% endif %}
+                <p> <b> Title: </b> {{post.title}} </p>
+                <p> <b> Description: </b> {{post.description}} </p>
+                <div class="author">
```

Comment:
Do we want the Like and Comment functionality for private posts?
TODO: redirect to private posts rather the the post posts page as it currently does by default